packages/sanity/package.json

```diff
@@ -264,6 +267,7 @@
     "log-symbols": "^2.2.0",
     "mendoza": "^3.0.0",
     "module-alias": "^2.2.2",
+    "moment": "^2.30.1",
```

Comment:
We're phasing out our usage of moment.js. Could you use [date-fns](https://date-fns.org/) instead? It's already a dependency.

Reply:
Removed moment.js in latest commit.